docs/api/exchange_rates.rst

```diff
@@ -0,0 +1,455 @@
+Exchange Rates API
+==================
+
+These endpoints will allow you to easily manage exchange rates. Base URI is `/api/v1/exchange-rates`.
+
+Exchange Rate structure
+-----------------------
+
+If you request an exchange rate via API, you will receive an object with the following fields:
+
++----------------+-------------------------------------------------------------------------------+
+| Field          | Description                                                                   |
++================+===============================================================================+
+| id             | Id of the exchange rate                                                       |
++----------------+-------------------------------------------------------------------------------+
+| ratio          | Exchange rate's ratio                                                         |
++----------------+-------------------------------------------------------------------------------+
+| sourceCurrency | :doc:`The currency object serialized with the default data </api/currencies>` |
++----------------+-------------------------------------------------------------------------------+
+| targetCurrency | :doc:`The currency object serialized with the default data </api/currencies>` |
```

Comment:
Now i'm conflicted on this one.
Should currency be capitalized or not? On one hand it an object of a currency, so no. Although on the other, it's the `Currency` object which acts like a real currency, so no clue. WDYT?